AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  AddNameAndTagsView.swift
```

Comment:
Name doesn't match the actual filename
(btw. this is the problem with those useless header comments in files, those need to be maintained and give zero value)